src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -208,7 +208,7 @@
     (->
       request
       (ring/set-authorized-name name)
-      (ring/set-authorized-authentic? (and
+      (ring/set-authorized-authenticated? (and
                                        (verified? request
```

Comment:
And if we're going back in for changes I suppose we should fix up the whitespace here too